etc/config-template.toml

```diff
@@ -135,6 +135,11 @@
 # Interval (s) to check region whether the data are consistent.
 # consistency-check-interval = 0
 
+[raftstore.coprocessor]
```

Comment:
IMO, you should define a coprocessor section, not in raftstore, then create the coprocessor outside and pass to raftstore.